src/types/Inputs.ts

```diff
@@ -0,0 +1,8 @@
+import { UseFormRegisterReturn } from "react-hook-form";
+
+export interface AuthInputProps {
+    register: UseFormRegisterReturn;
+    type: string;
+    placeholder:string;
+    maxLength?:number;
+}
```

Comment:
해당 input 같은 경우 maxLength나 placeholder와 같이 input에 기본적으로 들어가는 type은 저렇게 명시하지 말고 
```suggestion
export interface AuthInputProps extends React.HTMLAttributes<HTMLInputElement> {
    register: UseFormRegisterReturn;
}
```
위와 같이하면 불필요한 input element에 기본적인 type 선언을 하지 않고 이용할 수 있습니다